spec/polaris-catalog-apis/policy-management-apis.yaml

```diff
@@ -0,0 +1,579 @@
+                  $ref: '#/components/examples/NoSuchPolicyError'
+        419:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/AuthenticationTimeoutResponse'
+        503:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/ServiceUnavailableResponse'
+        5XX:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/ServerErrorResponse'
+
+    delete:
+      tags:
+        - Catalog API
+      summary: Delete a policy from the catalog
+      operationId: deletePolicy
+      description: Remove a policy from the catalog
+      responses:
+        204:
+          description: Success, no content
+        400:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/BadRequestErrorResponse'
+        401:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/UnauthorizedResponse'
+        403:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/ForbiddenResponse'
+        404:
+          description:
+            Not Found - NoSuchPolicyException, policy to get does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '../rest-catalog-open-api.yaml#/components/schemas/IcebergErrorResponse'
+              examples:
+                PolicyToDeleteDoesNotExist:
+                  $ref: '#/components/examples/NoSuchPolicyError'
+        419:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/AuthenticationTimeoutResponse'
+        503:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/ServiceUnavailableResponse'
+        5XX:
+          $ref: '../rest-catalog-open-api.yaml#/components/responses/ServerErrorResponse'
+
+  /v1/{prefix}/namespaces/{namespace}/policies/{policy}/mappings:
+    parameters:
+      - $ref: '../rest-catalog-open-api.yaml#/components/parameters/prefix'
+      - $ref: '../rest-catalog-open-api.yaml#/components/parameters/namespace'
+      - $ref: '#/components/parameters/policy'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Create a mapping between a policy and a resource entity
+      operationId: setPolicy
+      description: >
+        Create a mapping between a policy and a resource entity
+
+
+        Policy can be set at different levels, table, namespace, and catalog, and they will be inherited by lower level entities. Different levels are:
+
+        1. **Table-like level:** Policies specific to individual tables/views.
+
+        2. **Namespace level:** Policies that apply to all tables within a namespace.
+
+        3. **Catalog level:** Catalog-wide policies that ally across all tables within a namespace
+
+
+        Policy Inheritance Override rules
+
+        - Table-like-level policies override namespace and catalog policies
+
+        - Namespace-level policies override upper level namespace or catalog policies
+
+        - Overrides apply only to policies of the same type
```

Comment:
I'd ignore this part in the spec. The rules are not true for non-heritable policies. The rules may change for inheritable policies.